src/runtime/virtcontainers/pkg/annotations/annotations.go

```diff
@@ -24,6 +24,25 @@ const (
 	SandboxConfigPathKey = kataAnnotationsPrefix + "config_path"
 )
 
+// Annotations related to Confidential Containers (CoCo)
+const (
+	cocoAnnotationsPrefix      = "io.confidentialcontainers."
```

Comment:
As this is / will be part of the default Kata Containers configuration at some point, I'd prefer using the "io.katacontainers.*" prefix.  With this said, you don't have to add this new block for CoCo, but just increase / expand the one for kata, adding the `kataAnnotPreAttestationPrefix` and the  `kataAnnotSevPrefix`, and then keeping the PreAttestation / GuestPolicy in this const block here.

Reply:
Also, if we already have some tests for specific annotations (please, check the `annotations_tests.go` file, it'd be good to add some tests there to ensure we don't end up regressing on this at some point.

Reply:
> ...to ensure these newly introduced annotations will be present when Kata Containers is deployed via the CC Operator.

@fidencio I believe only `io.katacontainers.hypervisor.*` annotations are require to be explicitly enabled. That is, if we did `io.katacontainers.cc.*` the CC annotation would be enabled by default. Am I mistaken?

Reply:
Not really.  By default all the annotations but that one in the Makefile are disabled.

Reply:
That is what I thought as well. However, in doing this patch I was to test the new annotations without explicitly enabling them in the configuration file. I dug deeper and the only place I found where annotation filtering took place was here in [src/runtime/pkg/oci/untils.go:385](https://github.com/kata-containers/kata-containers/blob/8246de821ff327beb750b6239ac6c9cc6272f217/src/runtime/pkg/oci/utils.go#L385) (note the `KataAnnotationHypervisorPrefix` filter, which is what I refer to in my previous comment) . As far as I can tell, this code isn't executed with a launch via `kubectl apply -f `. In addition, I could not find another place where annotations are being filtered. I would expect to see something in `virtcontiners/sandbox.go` where the annotations are being applied. It's very possible I'm missing something here...

Reply:
Nice finding, that's a surprise for me.

So, that's good to know that we're fine and don't have to enable anything else.  Thanks for double-checking this @jimcadden!